skops/card/_model_card.py

```diff
@@ -373,6 +374,34 @@ def add_metrics(self, **kwargs: str) -> "Card":
             self._eval_results[metric] = value
         return self
 
+    def add_feature_importances(self, feature_importances, data) -> "Card":
+        """Visualize permutation importance.
+
+        Parameters
+        ----------
+        feature_importances : sklearn.utils.Bunch
+            Output of sklearn.inspection.permutation_importance()
+
+        data :
+
+        Returns
+        -------
+        self : object
+            Card object.
+        """
+        sorted_importances_idx = feature_importances.importances_mean.argsort()
+        fig, ax = plt.subplots()
+        ax.boxplot(
+            x=feature_importances.importances[sorted_importances_idx].T,
+            labels=data.columns[sorted_importances_idx],
+            vert=False,
+        )
+        ax.set_title("Permutation Importances")
+        ax.set_xlabel("Decrease in Accuracy Score")
```

Comment:
"Accuracy" is not quite correct, right? It can be any score, really. So maybe just "Decrease in Score" or "Deterioration of Score"?

Reply:
I edited this. the `plot_name` is shared by both the title on plot (to avoid ambiguity) and the name of the plot on the markdown so people can pass what they want.